apps/src/args.rs

```diff
@@ -219,6 +219,7 @@ Options:
   -H --header HEADER ...   Add a request header.
   -n --requests REQUESTS   Send the given number of identical requests [default: 1].
   --session-file PATH      File used to cache a TLS session for resumption.
+  --trust-origin-ca-pem <file>  Path to the pem file of the origin's CA, if not publicly trusted.
```

Comment:
```suggestion
  --trust-ca <file>  Path to a CA trust file in PEM format.
```

Reply:
Hmm, can it be `--trust-ca-pem` at least? It wasn't clear to me at first that only PEM files are accepted.

Or maybe it's possible to accept either .pem or .crt files? is that the difference between `load_verify_locations_from_file` and `use_certificate_chain_file`?

Reply:
Well, that's kind of what the help text is for ("...in PEM format.").

Also, no, `use_certificate_chain_file` is for configuring the full certificate chain that is presented to the peer, while `load_verify_locations_from_file` is for configuring the CA certificates used to _verify_ the peer's certificate.